[2023-11-02T00:00:11.941+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509
[2023-11-02T00:00:11.943+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city'
[2023-11-02T00:00:11.978+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509
[2023-11-02T00:00:11.979+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city'
[2023-11-02T00:00:12.019+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: col_api_etl.extract_numbeo_prices_from_city scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-02T00:00:12.031+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: col_api_etl.extract_numbeo_prices_from_city scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-02T00:00:12.031+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-02T00:00:12.047+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_numbeo_prices_from_city> on 2023-11-01 00:00:00+00:00
[2023-11-02T00:00:12.056+0000] {standard_task_runner.py:57} INFO - Started process 17273 to run task
[2023-11-02T00:00:12.056+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-11-02T00:00:12.061+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'col_api_etl', 'extract_numbeo_prices_from_city', 'scheduled__2023-11-01T00:00:00+00:00', '--job-id', '1164', '--raw', '--subdir', 'DAGS_FOLDER/col_api_etl.py', '--cfg-path', '/tmp/tmp1y7sfw_g']
[2023-11-02T00:00:12.064+0000] {standard_task_runner.py:85} INFO - Job 1164: Subtask extract_numbeo_prices_from_city
[2023-11-02T00:00:12.120+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509
[2023-11-02T00:00:12.121+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=col_api_etl/run_id=scheduled__2023-11-01T00:00:00+00:00/task_id=extract_numbeo_prices_from_city'
[2023-11-02T00:00:12.123+0000] {task_command.py:415} INFO - Running <TaskInstance: col_api_etl.extract_numbeo_prices_from_city scheduled__2023-11-01T00:00:00+00:00 [running]> on host 65ec58942685
[2023-11-02T00:00:12.230+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='col_api_etl' AIRFLOW_CTX_TASK_ID='extract_numbeo_prices_from_city' AIRFLOW_CTX_EXECUTION_DATE='2023-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-01T00:00:00+00:00'
[2023-11-02T00:00:12.784+0000] {clientserver.py:505} INFO - Error while sending or receiving.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer
[2023-11-02T00:00:12.786+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-11-02T00:00:12.786+0000] {java_gateway.py:1052} INFO - Exception while sending command.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
  File "/home/airflow/.local/lib/python3.8/site-packages/py4j/clientserver.py", line 506, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending
[2023-11-02T00:00:12.790+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-11-02T00:00:13.821+0000] {logging_mixin.py:151} INFO - Dubai
[2023-11-02T00:00:15.516+0000] {logging_mixin.py:151} INFO - Abu-Dhabi
[2023-11-02T00:00:17.142+0000] {logging_mixin.py:151} INFO - Sharjah
[2023-11-02T00:00:18.495+0000] {logging_mixin.py:151} INFO - Perth
[2023-11-02T00:00:20.149+0000] {logging_mixin.py:151} INFO - Melbourne
[2023-11-02T00:00:21.800+0000] {logging_mixin.py:151} INFO - Sydney
[2023-11-02T00:00:23.546+0000] {logging_mixin.py:151} INFO - Brisbane
[2023-11-02T00:00:25.216+0000] {logging_mixin.py:151} INFO - Hobart
[2023-11-02T00:00:26.524+0000] {logging_mixin.py:151} INFO - Adelaide
[2023-11-02T00:00:28.113+0000] {logging_mixin.py:151} INFO - Canberra
[2023-11-02T00:00:29.824+0000] {logging_mixin.py:151} INFO - Darwin
[2023-11-02T00:00:31.132+0000] {logging_mixin.py:151} INFO - Sarajevo
[2023-11-02T00:00:32.757+0000] {logging_mixin.py:151} INFO - Manama
[2023-11-02T00:00:34.178+0000] {logging_mixin.py:151} INFO - Sofia
[2023-11-02T00:00:36.136+0000] {logging_mixin.py:151} INFO - Sao-Paulo
[2023-11-02T00:00:37.893+0000] {logging_mixin.py:151} INFO - Rio-De-Janeiro
[2023-11-02T00:00:39.552+0000] {logging_mixin.py:151} INFO - Brasilia
[2023-11-02T00:00:40.854+0000] {logging_mixin.py:151} INFO - Toronto
[2023-11-02T00:00:43.009+0000] {logging_mixin.py:151} INFO - Montreal
[2023-11-02T00:00:44.957+0000] {logging_mixin.py:151} INFO - Vancouver
[2023-11-02T00:00:46.571+0000] {logging_mixin.py:151} INFO - Ottawa
[2023-11-02T00:00:48.368+0000] {logging_mixin.py:151} INFO - Calgary
[2023-11-02T00:00:49.973+0000] {logging_mixin.py:151} INFO - Edmonton
[2023-11-02T00:00:51.891+0000] {logging_mixin.py:151} INFO - Halifax
[2023-11-02T00:00:53.519+0000] {logging_mixin.py:151} INFO - Quebec-City
[2023-11-02T00:00:55.233+0000] {logging_mixin.py:151} INFO - Zurich
[2023-11-02T00:00:56.830+0000] {logging_mixin.py:151} INFO - Geneva
[2023-11-02T00:00:58.139+0000] {logging_mixin.py:151} INFO - Zug
[2023-11-02T00:00:59.728+0000] {logging_mixin.py:151} INFO - Basel
[2023-11-02T00:01:01.581+0000] {logging_mixin.py:151} INFO - Bern
[2023-11-02T00:01:03.333+0000] {logging_mixin.py:151} INFO - Lausanne
[2023-11-02T00:01:04.944+0000] {logging_mixin.py:151} INFO - Lugano
[2023-11-02T00:01:06.460+0000] {logging_mixin.py:151} INFO - Vaduz
[2023-11-02T00:01:07.826+0000] {logging_mixin.py:151} INFO - Santiago
[2023-11-02T00:01:09.582+0000] {logging_mixin.py:151} INFO - Chongqing
[2023-11-02T00:01:10.859+0000] {logging_mixin.py:151} INFO - Shanghai
[2023-11-02T00:01:12.527+0000] {logging_mixin.py:151} INFO - Beijing
[2023-11-02T00:01:14.565+0000] {logging_mixin.py:151} INFO - Chengdu
[2023-11-02T00:01:15.885+0000] {logging_mixin.py:151} INFO - Guangzhou
[2023-11-02T00:01:17.192+0000] {logging_mixin.py:151} INFO - Shenzhen
[2023-11-02T00:01:18.839+0000] {logging_mixin.py:151} INFO - Bogota
[2023-11-02T00:01:20.657+0000] {logging_mixin.py:151} INFO - Medellin
[2023-11-02T00:01:22.261+0000] {logging_mixin.py:151} INFO - Cali
[2023-11-02T00:01:23.904+0000] {logging_mixin.py:151} INFO - Cartagena
[2023-11-02T00:01:25.259+0000] {logging_mixin.py:151} INFO - San-Jose
[2023-11-02T00:01:27.075+0000] {logging_mixin.py:151} INFO - Prague
[2023-11-02T00:01:28.715+0000] {logging_mixin.py:151} INFO - Brno
[2023-11-02T00:01:30.351+0000] {logging_mixin.py:151} INFO - Ostrava
[2023-11-02T00:01:31.667+0000] {logging_mixin.py:151} INFO - Copenhagen
[2023-11-02T00:01:33.295+0000] {logging_mixin.py:151} INFO - Aarhus
[2023-11-02T00:01:35.698+0000] {logging_mixin.py:151} INFO - Aalborg
[2023-11-02T00:01:37.363+0000] {logging_mixin.py:151} INFO - Andorra-La-Vella
[2023-11-02T00:01:38.966+0000] {logging_mixin.py:151} INFO - Vienna
[2023-11-02T00:01:40.578+0000] {logging_mixin.py:151} INFO - Salzburg
[2023-11-02T00:01:42.221+0000] {logging_mixin.py:151} INFO - Graz
[2023-11-02T00:01:43.507+0000] {logging_mixin.py:151} INFO - Linz
[2023-11-02T00:01:45.094+0000] {logging_mixin.py:151} INFO - Innsbruck
[2023-11-02T00:01:46.388+0000] {logging_mixin.py:151} INFO - Brussels
[2023-11-02T00:01:48.039+0000] {logging_mixin.py:151} INFO - Antwerp
[2023-11-02T00:01:49.491+0000] {logging_mixin.py:151} INFO - Gent
[2023-11-02T00:01:51.092+0000] {logging_mixin.py:151} INFO - Liege
[2023-11-02T00:01:52.409+0000] {logging_mixin.py:151} INFO - Brugge
[2023-11-02T00:01:54.215+0000] {logging_mixin.py:151} INFO - Zagreb
[2023-11-02T00:01:55.824+0000] {logging_mixin.py:151} INFO - Split
[2023-11-02T00:01:57.481+0000] {logging_mixin.py:151} INFO - Dubrovnik
[2023-11-02T00:01:59.139+0000] {logging_mixin.py:151} INFO - Nicosia
[2023-11-02T00:02:00.466+0000] {logging_mixin.py:151} INFO - Tallinn
[2023-11-02T00:02:02.079+0000] {logging_mixin.py:151} INFO - Tartu
[2023-11-02T00:02:03.177+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-11-02T00:02:03.180+0000] {process_utils.py:131} INFO - Sending 15 to group 17273. PIDs of all processes in the group: [17273]
[2023-11-02T00:02:03.181+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 17273
[2023-11-02T00:02:03.181+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-11-02T00:02:03.233+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=17273, status='terminated', exitcode=0, started='00:00:11') (17273) terminated with exit code 0
